nemo_text_processing/g2p/modules.py

```diff
@@ -315,40 +316,29 @@ def __init__(
         else:
             self.use_chars = use_chars
 
+        # parse the dictionary, update it, and generate symbol set.
         if isinstance(phoneme_dict, str) or isinstance(phoneme_dict, pathlib.Path):
-            # Load phoneme_dict from file path
-            self.phoneme_dict, self.symbols = self._parse_as_cmu_dict(phoneme_dict)
+            # load the dictionary file where there may exist a digit suffix after a word, which
+            # represents the pronunciation variant of that word.
+            phoneme_dict_obj = defaultdict(list)
+            _alt_re = re.compile(r'\([0-9]+\)')
+            with open(phoneme_dict, "r") as fdict:
+                for line in fdict:
+                    if len(line) and ('A' <= line[0] <= 'Z' or line[0] == "'"):
+                        parts = line.strip().split()
+                        assert len(parts) == 2, f"Wrong format for the entry: {line.strip()}."
+                        word = re.sub(_alt_re, '', parts[0])
+                        phoneme_dict_obj[word].append(parts[1:])
```

Comment:
`parts` should only have two entries as per the previous assertion--change to `parts[1]`

Reply:
sure. will revise.